Database/Corrections/sodQuestFixes.lua

```diff
@@ -2286,6 +2286,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410023,
             [questKeys.zoneOrSort] = sortKeys.DRUID,
         },
+        [90133] = {
+            [questKeys.name] = "Slaughter from the Shadows",
+            [questKeys.startedBy] = {nil,{387466}},
```

Comment:
[Gnarlpine Cache](https://www.wowhead.com/classic/object=407734/gnarlpine-cache) and [Relic Coffer](https://www.wowhead.com/classic/object=404941/relic-coffer) also contains this rune. I suggest simply adding it to this fake quest directly and tweak the `objectiveText` a pit.

Also make sure to check the spawns of these chests. Most objects scraped for SoD have multiple spawns with just a really minor difference.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5434/commits/3c15c3a0379915f047793829164ebceafeb2005b